src/cryptography/key/secp256k1.ts

```diff
@@ -32,6 +36,22 @@ class KeyWrapper implements IAsymmetricKey {
     });
   }
 
+  public isValidPrivateKey(privateKey: Hex): boolean {
+    const privateKeyNumber = TypeUtils.bytesToNumber(TypeUtils.parseHexToArray(privateKey));
+    if (privateKeyNumber == 0n || privateKeyNumber >= secp.CURVE.n) {
+      return false;
+    }
+    return true;
+  }
+
+  public getKeyInPEM(key: Uint8Array, isPrivate: boolean): string {
+    if (isPrivate) {
+      return keyEncoder.encodePrivate(base16.encode(key), "raw", "pem");
```

Comment:
I think we also need `BEGIN` and `END` tag here, similar with ed25519

Reply:
The key-encode did that already